src/preact/amp-base-element.js

```diff
@@ -0,0 +1,3 @@
+import {PreactBaseElement} from './base-element';
+
+export class AmpPreactBaseElement extends PreactBaseElement {}
```

Comment:
nit: not necessary for this PR in particular, but the architecture we are deciding on right now should be written up in a jsdoc